datanode/src/updating.md

```diff
@@ -0,0 +1,11 @@
+# Updating version info
+
+When making changes, version information needs to be updated in several places.
```

Comment:
From storage_api, with formatting and spelling corrections:
**VERSION HISTORY - BRANCH.MAJOR.MINOR.BUILD**
Where BRANCH is the specific version for this branch, or numeric for the mainline branch,
changing only on new branches or fundamental changes to the mainline branch;
and MAJOR changes when there is major functionality change that either directly or eventually deprecates functionality in previous major versions;
and MINOR changes when there is a non-breaking API change (new fields or methods) and resets to 0 when a MAJOR change takes place;
and BUILD continuously increments with every release, regardless of other version number changes.

Reply:
I've moved the versioning information into updating.md and added a note pointing there from storage_api.py; will merge and deploy later today unless there are any other issues.